kodilla-advanced-tests/src/main/java/com/kodilla/parametrized_tests/homework/UserValidator.java

```diff
@@ -0,0 +1,33 @@
+package com.kodilla.parametrized_tests.homework;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class UserValidator {
+
+    public boolean validateUsername(String username) throws UserDoesntExist {
+     try {
+         return username.matches("^[a-zA-Z0-9._-]{3,}$");
+     } catch (Exception e) {
```

Comment:
przepraszam, a to skad sie wzielo?

Reply:
chciałam wykluczyć możliwość wpisania pustego username, dlatego dopisałam to do metody i utworzyłam klasę